// proto/sound_service.proto
syntax = "proto3";

package soundservice;

// Define the SoundService with a PlaySound RPC call
service SoundService {
    rpc PlaySound(SoundRequest) returns (SoundResponse);
}

// Request message containing audio data
message SoundRequest {
    string file_name = 1;
}

// Response message with playback status
message SoundResponse {
    string message = 1;
    bool success = 2;
}
